client/components/track-interactions/index.js

```diff
@@ -68,3 +58,17 @@ export default class TrackInteractions extends Component {
 		return cloneElement( child, props );
 	}
 }
+
+function trackInteraction( componentName, data ) {
```

Comment:
Worth noting that this is one of those actions whose placement in `state` is less obvious, since it doesn't correspond to a definite branch of state. If I *had* to pick a place, I might consider either `state.ui` or `state.ui.actionLog`. With that, I'll add that it's a good thing that actions can be detached from their impact on state — compare with a hypothetical `ACTION_LOG_ADD` action.